CalibForward/CTPPSPixelCalibration/BuildFile.xml

```diff
@@ -0,0 +1,9 @@
+<use   name="CondFormats/CTPPSReadoutObjects"/>
+<use   name="CondCore/DBOutputService"/>
+<use   name="FWCore/ServiceRegistry"/>
+<use   name="FWCore/Framework"/>
+<use name="CondFormats/DataRecord"/>
+<use name="DataFormats/CTPPSDetId"/>
+  <use   name="root"/>
```

Comment:
CondFormats/DataRecord is needed for the data records, the others are not. I removed them.